README.md

```diff
@@ -9,31 +9,15 @@ ipfx is a python 2/3 package for computing intrinsic cell features from electrop
 
 This software is designed for use in the Allen Institute for Brain Science electrophysiology data processing pipeline.
 
-## Quickstart:
-
-To run:
-
-```bash
```

Comment:
move this to quickstart in the docs

Reply:
Still need to reference files that actually exist.  The quickstart only lists 'pipeline_from_nwb.py' . This file does not exist.
https://github.com/AllenInstitute/ipfx/tree/master/ipfx/bin

I think the file that should be used is run_pipeline_from_nwb_file.py

Bad: python pipeline_from_nwb.py input_nwb_file outputdir
Good: python run_pipeline_from_nwb_file.py input_nwb_file outputdir

Reply:
revised as suggested, thanks.